recipes-debian/glibc/locales_debian.bb

```diff
@@ -0,0 +1,36 @@
+# base recipe: meta/recipes-core/glibc/glibc_2.29.bb
+# base branch: warrior
+# base commit: 5280cacb56cca9801448c9dab532450b8e5051bc
```

Comment:
This is your original recipe, so you don't need header description.

Reply:
The comment has been deleted.